test/jdk/java/awt/print/PrinterJob/PrintToFileTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 7179006
+ * @summary Confirm printing to file works.
+*/
+
+import java.io.File;
+import java.net.URI;
+import java.awt.Color;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterJob;
+import java.awt.print.PrinterException;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.Destination;
+
+public class PrintToFileTest implements Printable {
+
+    public PrintToFileTest() {}
+
+    public static void main(String[] args) throws Exception {
+        PrinterJob pj = PrinterJob.getPrinterJob();
+        if (pj.getPrintService() == null) {
+            System.out.println("No printers installed. Skipping test.");
+            return;
+        }
+        pj.setPrintable(new PrintToFileTest(), new PageFormat());
+        PrintRequestAttributeSet pSet = new HashPrintRequestAttributeSet();
+        File file = new File("./out.prn");
+        pSet.add(new Destination(file.toURI()));
+        pj.print(pSet);
+        if (!file.exists()) {
+             throw new RuntimeException("No file created");
+        }
```

Comment:
I guess we need to delete the file in finally block, if created.

Reply:
I meant to leave it. So we can look at it if we need to.